pom.xml

```diff
@@ -1585,6 +1585,14 @@
           <groupId>net.alchim31.maven</groupId>
           <artifactId>scala-maven-plugin</artifactId>
           <version>${plugin.scala.alchim31.version}</version>
+          <configuration>
```

Comment:
Thank you for solving the CI problem.

I would do the configuration of this plugin in the sub-projects. In most Scala subprojects there is already a custom compiler configuration and the settings from the parent pom.xml should have no effect there. It can be confusing if there is compiler configuration in two places.

Reply:
Oh! I honestly merged this setting after adding several places at first. But I also think it's good. I'll add similar settings in the proper places then.

Reply:
If we create a uniform configuration across all sub-modules, then I would definitely prefer a configuration in the parent pom. But I don't know if this is possible.

Reply:
I also don't think so. we should override parent configuration sometimes so I think it could be fine to have separate configuration for every sub-project.

Reply:
I think it is possible to put it in root pom's `pluginManagement` as default configuration, and sub module can customize its configuration.

Reply:
I removed redundant configuration only in this PR. I, however, feel like we'd better clean up scala configuration. @Reamer @zjffdu Maven option is only for building Zeppelin itself, not running it, so I think we could simplify the scala compile option. I created https://issues.apache.org/jira/browse/ZEPPELIN-5864